config/patches/pkg-config-lite/pkg-config-lite-0.28-1.config.guess.patch

```diff
@@ -0,0 +1,992 @@
+--- config.guess_OLD	2015-10-30 21:16:33.910969812 +0000
++++ config.guess	2015-10-30 21:16:39.771905250 +0000
+@@ -1,14 +1,12 @@
+ #! /bin/sh
+ # Attempt to guess a canonical system name.
+-#   Copyright (C) 1992, 1993, 1994, 1995, 1996, 1997, 1998, 1999,
+-#   2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010
+-#   Free Software Foundation, Inc.
++#   Copyright 1992-2015 Free Software Foundation, Inc.
+ 
+-timestamp='2009-12-30'
++timestamp='2015-10-21'
+ 
+ # This file is free software; you can redistribute it and/or modify it
+ # under the terms of the GNU General Public License as published by
+-# the Free Software Foundation; either version 2 of the License, or
++# the Free Software Foundation; either version 3 of the License, or
```

Comment:
My argument was for both of those...

And actually its most likely much more sustainable for us to bake modern autoconf into our build images (not in omnibus-software/omnibus-chef but into the omnibus cookbook or that omnibus package that @scotthain setup for build deps for omnibus itself).  Then instead of patching config.guess, we simply re-autoconf the software and regenerate configure+friends and run from there.

That is both less hacky, less patchy and won't show up as scary GPLv3 banners checked into our codebase, so its probably the right thing to do.  The patches to config.guess are being generated by this process by people making with the typey-typey to generate config.guess and then they create a patch and generate a PR and put it here.  That process seems broken and we should just automate it...